experiments/compositions/samples/AppTeam/README.md

```diff
@@ -4,15 +4,15 @@ For now this would only work in CC from a specific project.
 
 ## [Platform Admin] Create a Context object
 
-The first step is to create a context object in the namespace where compositions are installed.
+The first step is to create a context object in the namespace where AppTeam will be created.
 
 ```
 kubectl apply -f - <<EOF
 apiVersion: composition.google.com/v1alpha1
 kind: Context
 metadata:
   name: context
-  namespace: default
+  namespace: config-control
```

Comment:
fine for now but we need to come up with guidance/best practices on laying out namespaces and permissions.